plasmapy/particles/decorators.py

```diff
@@ -744,8 +744,7 @@ def particle_input(
           class SomeClass:
               @particle_input
               @validate_quantities
-              def instance_method(self, particle: ParticleLike, B: u.Quantity[u.T]):
-                  ...
+              def instance_method(self, particle: ParticleLike, B: u.Quantity[u.T]): ...
```

Comment:
When I run the `blacken-docs` hook for `pre-commit` locally on my computer, I get the `...` on the following line. When I do `pre-commit.ci autofix`, it puts them on the same line.  Hm...